src/ai/backend/manager/models/kernel.py

```diff
@@ -1358,7 +1358,12 @@ def parse_row(cls, ctx: GraphQueryContext, row: Row) -> Mapping[str, Any]:
     def from_row(cls, context: GraphQueryContext, row: Row) -> Optional[LegacyComputeSession]:
         if row is None:
             return None
-        props = cls.parse_row(context, row)
+        props = dict(cls.parse_row(context, row))
```

Comment:
why do we need to convert twice here? the return type of `parse_row()` is Mapping.

Reply:
The dict type is mutable, but attempting to modify a Mapping type will result in a type error.

Reply:
Then we should update the `parse_row()` method to not resolve `status_data` and `vfolder_mounts` fields, not update the result of it